tests/libcxx/expected_results.txt

```diff
@@ -616,6 +613,10 @@ std/utilities/utility/mem.res/mem.res.monotonic.buffer/mem.res.monotonic.buffer.
 # we'd need to reverse the order of the _Mtx_unlock() and _Cnd_broadcast() calls in xnotify.cpp.
 std/thread/thread.condition/notify_all_at_thread_exit_lwg3343.pass.cpp SKIPPED
 
+# LWG-4266 "`layout_stride::mapping` should treat empty mappings as exhaustive"
+std/containers/views/mdspan/layout_stride/is_exhaustive_corner_case.pass.cpp FAIL
+std/containers/views/mdspan/layout_stride/properties.pass.cpp FAIL
+

```

Comment:
This is being placed in "MISSING LWG ISSUE RESOLUTIONS" but that section is when we're missing a resolution. I believe this should be categorized as "libc++ is missing this resolution".